main-repo-triage/strings.json

```
@@ -10,5 +10,6 @@
     "too_few_logs": "You have not provided a sufficient amount of logs. Please provide every line during the time of the event *plus 10 lines before and after*.",
     "no_ffmpeg_log": "You have not provided any ffmpeg logs. They should be included if transcoding or remuxing.",
     "invalid_ffmpeg_log": "You have not provided the ffmpeg log unaltered from the server. Please provide a valid ffmpeg log.",
-    "not_jellyfin_ffmpeg": "You are not using our custom ffmpeg fork. Compatibility is not gauranteed."
+    "not_jellyfin_ffmpeg": "You are not using our custom ffmpeg fork. Compatibility is not gauranteed.",
+    "using_microsoft_iis": "You are using IIS, which is an unsupported environment."
```

i would really expand on this and keep it in line with the logline:

```suggestion
    "using_microsoft_iis": "UNSUPPORTED HOSTING ENVIORMENT Microsoft Internet Information Services. You are using the IIS to host jellyfin which is an unsupported and untested feature. We will close any bug that is assumed to be rooted in this Hosting Environment"
```

---

The log line isn't finalized in the IIS PR yet, will copy that over when its merged